app/src/main/java/com/schibsted/nmp/warpapp/MainActivity.kt

```diff
@@ -10,7 +10,17 @@ class MainActivity : ComponentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        //setContentView(R.layout.main_activity_layout)
+        /*
```

Comment:
Comment can be removed now? :)

Reply:
It's always there :D I use this layout to text the xml versions of the components